src/screens/ChatDetailPage/index.js

```diff
@@ -208,7 +208,7 @@ const CustomDateHeader = () => null;
 export default withInteractionsManaged(ChatDetailPage);
 const styles = StyleSheet.create({
   date: {
-    color: COLORS.white,
+    color: '#fff',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been changed from a constant `COLORS.white` to a hardcoded value `'#fff'`. This could potentially lead to inconsistencies if the color needs to be updated in the future. It's generally better to use constants for such values to ensure consistency across the application.
```diff
-    color: '#fff',
+    color: COLORS.white,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->